builds/dumbo/scripts/controllers/login.js

```diff
@@ -1,168 +1,216 @@
-angular.module('dumboApp')
-.controller('LoginCtrl', function ($scope, userDataService, SweetAlert, userService,$routeParams, $location, ngToast) {
-    var id = $routeParams.id;
-    var key = $routeParams.key;
-    var action = $routeParams.action;
-    var loginType = $routeParams.loginType;
-    $scope.init = function() {
-        //setup page actions
-        if (loginType=="newuser"){
-            $scope.newuser = true;
-        } else {
-            $scope.newuser= false;
-        }
-        if (id && key && action=="confirm"){
-            userDataService.confirmAccount(id,key).then(function(data){
-                SweetAlert.swal({title: "Welcome!",
-                text: "Your account is confirmed. Please log in.",
-                timer: 2000,
-                showConfirmButton: false });
-
-            }, function(data){
-                SweetAlert.swal({title: "Sorry",
-                text: "Please check your email, your account was not confirmed",
-                timer: 2000,
-                showConfirmButton: false });
-            })
-        }
-        if (id && key && action=="recover"){
-            $scope.resetPassword();
-        }
-    };
+angular.module('dumboApp').controller('LoginCtrl',
```

Comment:
I refactored the shit out of this file. I couldn't work on it with the indentation/spacing it had, so I reworked it.